src/main/groovy/wooga/gradle/paket/unity/PaketUpmPackageSpec.groovy

```diff
@@ -0,0 +1,68 @@
+package wooga.gradle.paket.unity
+
+import com.wooga.gradle.BaseSpec
+import org.gradle.api.provider.MapProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.provider.Provider
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Optional
+
+trait PaketUpmPackageSpec implements BaseSpec {
+
+    /**
+     * Map [<paket_package_name>: <upm_manifest_contents>] that overrides the contents of generated package.json files in UPM package mode.
+     */
+    private final MapProperty<String, Map<String, Object>> paketUpmPackageManifest = objects.mapProperty(String, Map)
+
+    @Input
+    @Optional
+    MapProperty<String, Map<String, Object>> getPaketUpmPackageManifests() {
+        return paketUpmPackageManifest
+    }
```

Comment:
There's a typo, an `s` at the end?

Reply:
Its not a typo, there may be many manifests. Its a map of [paket_name: upm manifest] after all.